hack/generator/pkg/codegen/pipeline_mark_storage_version.go

```diff
@@ -72,6 +72,15 @@ func groupResourcesByVersion(types astmodel.Types) (map[unversionedName][]astmod
 	result := make(map[unversionedName][]astmodel.TypeDefinition)
 
 	for _, def := range types {
+
+		// TODO: Remove this
```

Comment:
If you have a bug tracking it, reference the bug here?

If not, don't worry about it.

Reply:
Works for me.

Reply:
Don't have one.